index.html

```diff
@@ -8,22 +8,35 @@
     <link rel="stylesheet" href="css/styles.css" type="text/css" />
   </head>
   <body>
-    <h1>Learn to code by watching others</h1>
 
-    <p>See how experienced developers solve problems in real-time. Watching scripted tutorials is great, 
-    but understanding how developers think is invaluable. </p>
-  
-    <div>Try it free 7 days then $20/mo. thereafter</div>
-    <div>
-      First Name 
-      Last Name 
-      Email Address
-      Password
-  
-      <div>Claim your free trial </div>
+    <div class="text-blurb">
+      <h1>Learn to code by watching others</h1>
+      <p>See how experienced developers solve problems in real-time. Watching scripted tutorials is great, 
+      but understanding how developers think is invaluable. </p>
     </div>
-  
-    By clicking the button, you are agreeing to our Terms and Services
+
+    <div class="try-it">
+      <p><strong>Try it free 7 days</strong> then $20/mo. thereafter</p>
+    </div>
+
+    <div class="sign-up">
+      <div class="sign-boxes">
+        <input type="text" placeholder="First Name" />
+        <input type="text" placeholder="Last Name" />
+        <input type="text" placeholder="Email Address" />
```

Comment:
make this `type="email"`. While email and text fields may look the same, by using the email field the browser will automatically check if it's a valid email in the field when you try and submit the form.